eng/pipelines/branch-cleanup.yml

```
@@ -30,14 +44,31 @@ jobs:
           displayName: Clean Up Sync Common Branches
           condition: succeededOrFailed()
           continueOnError: true
+          inputs:
+            pwsh: true
+            workingDirectory: $(System.DefaultWorkingDirectory)
+            filePath: $(System.DefaultWorkingDirectory)/eng/common/scripts/Delete-RemoteBranches.ps1
+            arguments: >
+              -RepoId ${{ repo }} 
+              -CentralRepoId "Azure/azure-sdk-tools"\
+              -CentralPRRegex "^sync-eng/common.*[^-]*-(?<PrNumber>\d+)$"
```

```suggestion
              -CentralPRRegex "^sync-eng/common.*-(?<PrNumber>\d+)$"
```

I don't think the `[^-]*` is needed.